docs/en/observability/synthetics-run-test.asciidoc

```diff
@@ -0,0 +1,205 @@
+[[synthetic-run-tests]]
+= Run synthetic tests
+
+++++
+<titleabbrev>Run synthetic tests</titleabbrev>
+++++
+
+There are two ways to run synthetic tests:
+
+* If you want to create a single journey and manage it in isolation from monitors containing other journeys,
+<<synthetics-inline-journey,use an inline journey>>.
+Inline journeys can be more efficient, but have some limitations like not supporting dependencies or multiple journeys.
+* If you have multiple journeys, rely on dependencies, or your tests need to live with your
+application code, <<synthetics-test-suite,use a test suite>>.
+
+[discrete]
+[[synthetics-inline-journey]]
+== Use an inline journey
+
+The easiest way to run a synthetic test is by creating an inline journey.
+The `journey` keyword isn't required, and access to variables like `page` and `params` is automatic.
+You cannot `import` any dependencies using inline scripts.
+
+To test an inline example locally, change into your test's directory
+and pipe the file contents to the `npx @elastic/synthetics` command.
+
+For example, create a `sample.js` file containing steps:
```

Comment:
```suggestion
Assume, for example, that your inline monitor contains the following code:
```

IMO it's important that users know that an inline journey refers to the content of the journey, not to a specific file, which is why I'm suggesting this.